jdi-light-mobile/src/main/java/com/epam/jdi/mobile/driver/get/DriverData.java

```diff
@@ -157,95 +222,92 @@ public static void setUp(String name, JAction action) {
-            firefoxProfile.setPreference("browser.helperApps.alwaysAsk.force", false);
-            firefoxProfile.setPreference("browser.helperApps.neverAsk.saveToDisk", "application/xls;text/csv;text/plain");
-            firefoxProfile.setPreference("browser.download.dir", DOWNLOADS_DIR);
-            firefoxProfile.setPreference("print.always_print_silent", "true");
-            firefoxProfile.setPreference("print.show_print_progress", "false");
-            firefoxProfile.setPreference("browser.startup.homepage", "about:blank");
-            firefoxProfile.setPreference("startup.homepage_welcome_url", "about:blank");
-            firefoxProfile.setPreference("startup.homepage_welcome_url.additional", "about:blank");
-            firefoxProfile.setPreference("network.http.phishy-userpass-length", 255);
-        });
-        setUp("Firefox: PageLoadStrategy:" + PAGE_LOAD_STRATEGY,
-            () -> cap.setPageLoadStrategy(PAGE_LOAD_STRATEGY));
-        setUp("Firefox: ACCEPT_SSL_CERTS: true",
-            () -> cap.setCapability(ACCEPT_SSL_CERTS, true));
-        setUp("Firefox: UNEXPECTED_ALERT_BEHAVIOR, ACCEPT",
-            () -> cap.setCapability(UNEXPECTED_ALERT_BEHAVIOR, ACCEPT));
-        setUp("Firefox: Firefox Profile",
-            () -> cap.setProfile(firefoxProfile));
-        // Capabilities from settings
-        CAPABILITIES_FOR_FF.forEach(cap::setCapability);
-    };
-
-    public static JAction1<InternetExplorerOptions> IE_OPTIONS = cap -> {
-        setUp("IE: introduceFlakinessByIgnoringSecurityDomains",
-            cap::introduceFlakinessByIgnoringSecurityDomains);
-        setUp("ignoreZoomSettings",
-            cap::ignoreZoomSettings);
-        setUp("IE: requireWindowFocus:true",
-            () -> cap.setCapability("requireWindowFocus", true));
-        setUp("IE: PageLoadStrategy:" + PAGE_LOAD_STRATEGY,
-            () -> cap.setPageLoadStrategy(PAGE_LOAD_STRATEGY));
-        setUp("IE: takeFullPageScreenshot",
-            cap::takeFullPageScreenshot);
-        setUp("IE: ACCEPT_SSL_CERTS: true",
-            () -> cap.setCapability(ACCEPT_SSL_CERTS, true));
-        setUp("IE: destructivelyEnsureCleanSession",
-            cap::destructivelyEnsureCleanSession);
-        setUp("IE: UNEXPECTED_ALERT_BEHAVIOR: ACCEPT)",
-            () -> cap.setCapability(UNEXPECTED_ALERT_BEHAVIOR, ACCEPT));
-        setUp("IE: SUPPORTS_JAVASCRIPT",
-            () -> cap.is(SUPPORTS_JAVASCRIPT));
-        setUp("IE: ACCEPT_SSL_CERTS: true",
-            () -> cap.setCapability(ACCEPT_SSL_CERTS, true));
-        // Capabilities from settings
-        CAPABILITIES_FOR_IE.forEach(cap::setCapability);
-    };
-    public static JAction1<EdgeOptions> EDGE_OPTIONS = cap -> {
-
-    };
-    public static JAction1<OperaOptions> OPERA_OPTIONS = cap -> {
-
-    };
+    private static WebDriver maximizeScreen(WebDriver driver) {
+        try {
+            switch (getOs()) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Switch statements should have a default label](https://app.codacy.com/manual/42_/jdi-light/pullRequest?prid=4973196)